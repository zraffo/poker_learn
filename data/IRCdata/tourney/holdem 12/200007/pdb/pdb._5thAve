_5thAve   964660524 12  7 f   -     -     -        10000    0    0 
_5thAve   964660567 12  6 f   -     -     -        10000    0    0 
_5thAve   964660620 12  5 f   -     -     -        10000    0    0 
_5thAve   964660695 12  4 f   -     -     -        10000    0    0 
_5thAve   964660746 12  3 c   kf    -     -        10000   50    0 
_5thAve   964660823 11  2 Bf  -     -     -         9950   50    0 
_5thAve   964660885 11  1 Bc  kf    -     -         9900  225    0 
_5thAve   964660930 11 11 f   -     -     -         9675    0    0 
_5thAve   964661015 11 10 c   b     -     -         9675  700 1200 
_5thAve   964661116 11  9 f   -     -     -        10175    0    0 
_5thAve   964661182 11  8 c   f     -     -        10175  100    0 
_5thAve   964661233 11  7 c   k     r     -        10075 2300 3300 
_5thAve   964661314 11  6 cc  b     c     -        11075 9325 22150 Jc 8c
_5thAve   964661428 10  4 c   r     -     -        23900 5000 7200 
_5thAve   964661486 10  3 f   -     -     -        26100    0    0 
_5thAve   964661547 10  2 Brc -     -     -        26100 7775    0 Ks Kd
_5thAve   964661573 10  1 Bf  -     -     -        18325  100    0 
_5thAve   964661622 10 10 c   c     c     -        18225 6975    0 8c 8s
_5thAve   964661678 10  9 c   kc    kc    kcA      11250 11250 24100 7d 6s
_5thAve   964661736 10  8 c   b     -     -        24100 9600 20925 6d Ac
_5thAve   964661812  9  6 cf  -     -     -        35425  900    0 
_5thAve   964661850  8  5 c   f     -     -        34525  900    0 
_5thAve   964661907  8  4 c   f     -     -        33625  900    0 
_5thAve   964661936  8  3 f   -     -     -        32725    0    0 
_5thAve   964661972  8  2 Bk  c     c     c        32725 21625 21625 6h 9s
_5thAve   964662018  8  1 Bf  -     -     -        32725  200    0 
_5thAve   964662076  8  8 f   -     -     -        32525    0    0 
_5thAve   964662107  8  7 f   -     -     -        32525    0    0 
_5thAve   964662123  8  6 f   -     -     -        32525    0    0 
_5thAve   964662147  8  5 f   -     -     -        32525    0    0 
_5thAve   964662166  8  4 f   -     -     -        32525    0    0 
_5thAve   964662210  8  3 f   -     -     -        32525    0    0 
_5thAve   964662270  8  2 Bc  kf    -     -        32525 2800    0 
_5thAve   964662291  8  1 Bc  kf    -     -        29725  800    0 
_5thAve   964662324  8  8 f   -     -     -        28925    0    0 
_5thAve   964662350  8  7 c   b     -     -        28925 3600 5600 
_5thAve   964662384  8  6 cc  kf    -     -        30925 2800    0 
_5thAve   964662413  8  5 f   -     -     -        28125    0    0 
_5thAve   964662458  8  4 f   -     -     -        28125    0    0 
_5thAve   964662493  8  3 f   -     -     -        28125    0    0 
_5thAve   964662509  8  2 Bf  -     -     -        28125 1600    0 
_5thAve   964662531  8  1 Bf  -     -     -        26525  800    0 
_5thAve   964662573  8  8 f   -     -     -        25725    0    0 
_5thAve   964662603  8  7 cc  -     -     -        25725 19950    0 Ah 9c
_5thAve   964662629  7  6 c   f     -     -         5775 1600    0 
_5thAve   964662681  7  5 c   bA    -     -         4175 4175    0 3h Ah
_5thAve   964921272 12  4 f   -     -     -        10000    0    0 
_5thAve   964921357 12  3 f   -     -     -        10000    0    0 
_5thAve   964921409 12  2 Bf  -     -     -        10000   50    0 
_5thAve   964921450 12  1 Bf  -     -     -         9950   25    0 
_5thAve   964921524 12 12 f   -     -     -         9925    0    0 
_5thAve   964921587 12 11 f   -     -     -         9925    0    0 
_5thAve   964921642 12 10 f   -     -     -         9925    0    0 
_5thAve   964921665 12  9 f   -     -     -         9925    0    0 
_5thAve   964921741 12  8 c   b     -     -         9925  750 1300 
_5thAve   964921806 12  7 c   kf    -     -        10475  100    0 
_5thAve   964921907 12  6 f   -     -     -        10375    0    0 
_5thAve   964921956 12  5 cc  -     -     -        10375 8125    0 Tc 9s
_5thAve   964922001 12  4 f   -     -     -         2250    0    0 
_5thAve   964922027 11  2 f   -     -     -         2250    0    0 
_5thAve   964922100 11  2 Bf  -     -     -         2250  200    0 
_5thAve   964922174 11  1 Br  bA    -     -         2050 2050    0 9c As
_5thAve   964970303 14  5 cc  k     bcA   -         5000 5000 16050 Jc Kc
_5thAve   964970408 12  4 f   -     -     -        16050    0    0 
_5thAve   964970480 12  3 cc  kc    -     -        16050 4975    0 Tc Qh
_5thAve   964970563 11  2 Bf  -     -     -        11075   50    0 
_5thAve   964970610 10  1 Bc  k     kf    -        11025  100    0 
_5thAve   964970662 10 10 f   -     -     -        10925    0    0 
_5thAve   964970672 10  9 c   k     f     -        10925  100    0 
_5thAve   964970738 10  8 f   -     -     -        10825    0    0 
_5thAve   964970762 10  7 c   k     k     f        10825  100    0 
_5thAve   964970821 10  6 cc  kc    -     -        10725 3675    0 7s 8c
_5thAve   964970859 10  5 f   -     -     -         7050    0    0 
_5thAve   964970882 10  4 cc  kf    -     -         7050  550    0 
_5thAve   964970953 10  3 f   -     -     -         6500    0    0 
_5thAve   964971035 10  2 Bk  b     b     -         6500 1800 2400 
_5thAve   964971081 10  1 Bc  b     k     k         7100 1800    0 Kc 8h
_5thAve   964971118  9  9 f   -     -     -         5300    0    0 
_5thAve   964971162  9  8 f   -     -     -         5300    0    0 
_5thAve   964971194  8  6 f   -     -     -         5300    0    0 
_5thAve   964971240  8  5 f   -     -     -         5300    0    0 
_5thAve   964971273  8  4 f   -     -     -         5300    0    0 
_5thAve   964971280  8  3 cc  kcA   -     -         5300 5300 13000 6d 2d
_5thAve   964971305  8  2 Bc  -     -     -        13000 2050    0 Qh 5h
_5thAve   964971334  8  1 Bf  -     -     -        10950  200    0 
_5thAve   964971344  8  8 f   -     -     -        10750    0    0 
_5thAve   964971367  8  7 cc  k     k     k        10750 1325    0 9c 2c
_5thAve   964971402  8  6 c   c     c     c         9425 4300 9000 Jc Td
_5thAve   964971440  7  5 c   b     kc    -        14125 4375 9350 5c 8c
_5thAve   964971470  6  4 c   k     c     b        19100 4400 6000 
_5thAve   964971509  6  3 f   -     -     -        20700    0    0 
_5thAve   964971521  6  2 Bf  -     -     -        20700  400    0 
_5thAve   964971533  6  1 Bf  -     -     -        20300  200    0 
_5thAve   964971547  6  6 c   f     -     -        20100 1400    0 
_5thAve   964971567  5  4 c   b     k     b        18700 3600 4800 
_5thAve   964971604  5  4 f   -     -     -        19900    0    0 
_5thAve   964971611  5  3 c   k     k     k        19900  800    0 7d 2d
_5thAve   964971629  5  2 Bc  bcA   -     -        19100 19100 38600 9d Ts
_5thAve   964971664  5  1 Bc  k     b     -        38600 2400 3200 
_5thAve   964971682  5  5 f   -     -     -        39400    0    0 
_5thAve   964971691  5  4 f   -     -     -        39400    0    0 
_5thAve   964971697  5  3 c   b     -     -        39400 3200 4800 
_5thAve   964971719  5  2 Bf  -     -     -        41000  800    0 
_5thAve   964971730  5  1 Bc  b     -     -        40200 38300 28425 7c 7s
_5thAve   964971755  3  3 c   k     k     k        30325  800 2000 7c Qh
_5thAve   964971776  3  2 Bk  b     -     -        31525 2400 3200 
_5thAve   964971792  3  1 Bc  k     k     kf       32325 2800    0 
_5thAve   964971812  3  3 c   b     -     -        29525 2800 4000 
_5thAve   964971823  3  2 Bf  -     -     -        30725  800    0 
_5thAve   964971827  3  1 Brc -     -     -        29925 4100 8200 Kh Ks
_5thAve   964971835  2  2 r   -     -     -        34025 2400 3200 
_5thAve   964971842  2  2 Bk  f     -     -        34825  800    0 
_5thAve   964971847  2  1 Bf  -     -     -        34025  400    0 
_5thAve   964971850  2  2 Bf  -     -     -        33625  800    0 
_5thAve   964971853  2  1 Bcc kc    k     kc       32825 9600    0 As 9c
_5thAve   964971879  2  2 Bc  k     k     k        23225 2400 4800 Kh Th
_5thAve   964971894  2  1 Bcc b     k     b        25625 21600 28800 
_5thAve   964971939  2  2 B   -     -     -        32825 1600 2400 
_5thAve   964971942  2  1 Bf  -     -     -        33625  800    0 
_5thAve   964971944  2  2 Bc  c     k     k        32825 14400 28800 2d 9d
_5thAve   964971965  2  1 Br  -     -     -        47225 4800 6400 
_5thAve   964971973  2  2 Br  r     -     -        48825 43200 64375 Ks Kh
_5thAve   964973424  6  1 Bf  -     -     -        10000   50    0 
_5thAve   964973515  6  6 f   -     -     -         9950    0    0 
_5thAve   964973521  6  5 f   -     -     -         9950    0    0 
_5thAve   964973540  6  4 f   -     -     -         9950    0    0 
_5thAve   964973551  6  3 f   -     -     -         9950    0    0 
_5thAve   964973553  6  2 B   -     -     -         9950  100  150 
_5thAve   964973556  6  1 Bf  -     -     -        10000   50    0 
_5thAve   964973561  6  6 f   -     -     -         9950    0    0 
_5thAve   964973564  6  5 f   -     -     -         9950    0    0 
_5thAve   964973573  6  4 f   -     -     -         9950    0    0 
_5thAve   964973578  6  3 f   -     -     -         9950    0    0 
_5thAve   964973579  6  2 Bk  f     -     -         9950  100    0 
_5thAve   964973582  6  1 Bf  -     -     -         9850   50    0 
_5thAve   964973589  6  6 f   -     -     -         9800    0    0 
_5thAve   964973596  6  5 f   -     -     -         9800    0    0 
_5thAve   964973616  6  4 f   -     -     -         9800    0    0 
_5thAve   964973628  6  3 f   -     -     -         9800    0    0 
_5thAve   964973633  6  2 Bf  -     -     -         9800  100    0 
_5thAve   964973639  6  1 Bf  -     -     -         9700   50    0 
_5thAve   964973651  6  6 f   -     -     -         9650    0    0 
_5thAve   964973672  6  5 f   -     -     -         9650    0    0 
_5thAve   964973679  6  4 f   -     -     -         9650    0    0 
_5thAve   964973698  6  3 f   -     -     -         9650    0    0 
_5thAve   964973713  6  2 Bk  f     -     -         9650  100    0 
_5thAve   964973722  6  1 Bf  -     -     -         9550   50    0 
_5thAve   964973730  6  6 f   -     -     -         9500    0    0 
_5thAve   964973738  6  5 f   -     -     -         9500    0    0 
_5thAve   964973760  6  4 f   -     -     -         9500    0    0 
_5thAve   964973776  6  3 f   -     -     -         9500    0    0 
_5thAve   964973788  5  2 Bf  -     -     -         9500  200    0 
_5thAve   964973806  5  1 Bf  -     -     -         9300  100    0 
_5thAve   964973817  5  5 f   -     -     -         9200    0    0 
_5thAve   964973823  5  4 f   -     -     -         9200    0    0 
_5thAve   964973829  5  3 f   -     -     -         9200    0    0 
_5thAve   964973834  5  2 Bk  f     -     -         9200  200    0 
_5thAve   964973852  5  1 Bf  -     -     -         9000  100    0 
_5thAve   964973880  5  5 f   -     -     -         8900    0    0 
_5thAve   964973888  5  4 f   -     -     -         8900    0    0 
_5thAve   964973894  5  3 f   -     -     -         8900    0    0 
_5thAve   964973917  5  2 Bf  -     -     -         8900  200    0 
_5thAve   964973925  5  1 Bf  -     -     -         8700  100    0 
_5thAve   964973936  5  5 f   -     -     -         8600    0    0 
_5thAve   964973941  5  4 f   -     -     -         8600    0    0 
_5thAve   964973946  5  3 f   -     -     -         8600    0    0 
_5thAve   964973960  5  2 Bf  -     -     -         8600  200    0 
_5thAve   964973969  5  1 Bf  -     -     -         8400  100    0 
_5thAve   964973977  5  5 f   -     -     -         8300    0    0 
_5thAve   964973988  5  4 f   -     -     -         8300    0    0 
_5thAve   964974019  5  3 f   -     -     -         8300    0    0 
_5thAve   964974032  5  2 Bf  -     -     -         8300  200    0 
_5thAve   964974039  5  1 Bf  -     -     -         8100  100    0 
_5thAve   964974045  5  5 f   -     -     -         8000    0    0 
_5thAve   964974056  5  4 f   -     -     -         8000    0    0 
_5thAve   964974097  5  3 f   -     -     -         8000    0    0 
_5thAve   964974120  5  2 Bf  -     -     -         8000  400    0 
_5thAve   964974141  5  1 Bf  -     -     -         7600  200    0 
_5thAve   964974168  5  5 f   -     -     -         7400    0    0 
_5thAve   964974184  4  3 f   -     -     -         7400    0    0 
_5thAve   964974195  4  2 Bf  -     -     -         7400  400    0 
_5thAve   964974216  4  1 Bf  -     -     -         7000  200    0 
_5thAve   964974221  4  4 f   -     -     -         6800    0    0 
_5thAve   964974228  4  3 f   -     -     -         6800    0    0 
_5thAve   964974244  4  2 Bk  kf    -     -         6800  400    0 
_5thAve   964974271  4  1 Bf  -     -     -         6400  200    0 
_5thAve   964974276  4  4 f   -     -     -         6200    0    0 
_5thAve   964974282  4  3 f   -     -     -         6200    0    0 
_5thAve   964974302  4  2 Bf  -     -     -         6200  400    0 
_5thAve   964974312  4  1 Bf  -     -     -         5800  200    0 
_5thAve   964974316  4  4 f   -     -     -         5600    0    0 
_5thAve   964974322  4  3 f   -     -     -         5600    0    0 
_5thAve   964974343  4  2 Bk  kf    -     -         5600  400    0 
_5thAve   964974380  4  1 Bf  -     -     -         5200  200    0 
_5thAve   964974385  4  4 f   -     -     -         5000    0    0 
_5thAve   964974402  4  3 f   -     -     -         5000    0    0 
_5thAve   964974427  4  2 Bk  kf    -     -         5000  400    0 
_5thAve   964974441  4  1 Bf  -     -     -         4600  400    0 
_5thAve   964974445  4  4 f   -     -     -         4200    0    0 
_5thAve   964974485  4  3 f   -     -     -         4200    0    0 
_5thAve   964974508  4  2 Bf  -     -     -         4200  800    0 
_5thAve   964974512  4  1 Bf  -     -     -         3400  400    0 
_5thAve   964974516  4  4 f   -     -     -         3000    0    0 
_5thAve   964974538  4  3 f   -     -     -         3000    0    0 
_5thAve   964974547  4  2 Bf  -     -     -         3000  800    0 
_5thAve   964974554  4  1 Bf  -     -     -         2200  400    0 
_5thAve   964974560  4  4 f   -     -     -         1800    0    0 
_5thAve   964974578  4  3 f   -     -     -         1800    0    0 
_5thAve   964974589  4  2 Bk  k     k     k         1800  800    0 Kc 4s
_5thAve   964974611  4  1 Bf  -     -     -         1000  400    0 
_5thAve   964974637  4  4 f   -     -     -          600    0    0 
_5thAve   964974644  4  3 f   -     -     -          600    0    0 
_5thAve   964974649  4  2 BA  -     -     -          600  600 1600 Kh 6s
_5thAve   964974658  4  1 Bf  -     -     -         1600  400    0 
_5thAve   964974668  4  4 f   -     -     -         1200    0    0 
_5thAve   964974686  4  3 f   -     -     -         1200    0    0 
_5thAve   964974708  4  2 Bk  kf    -     -         1200  800    0 
_5thAve   964974719  4  1 BA  -     -     -          400  400 1200 Kd Js
_5thAve   964974747  4  4 f   -     -     -         1200    0    0 
_5thAve   964974755  4  3 f   -     -     -         1200    0    0 
_5thAve   964974774  3  1 BA  -     -     -         1200 1200    0 9s 5s
