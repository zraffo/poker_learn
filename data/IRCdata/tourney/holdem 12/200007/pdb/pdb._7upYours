_7upYours 964520627  8  4 f   -     -     -        10000    0    0 
_7upYours 964520636  8  3 f   -     -     -        10000    0    0 
_7upYours 964520669  8  2 Bf  -     -     -        10000   50    0 
_7upYours 964520714  8  1 Bf  -     -     -         9950   25    0 
_7upYours 964520731  8  8 f   -     -     -         9925    0    0 
_7upYours 964520749  8  7 f   -     -     -         9925    0    0 
_7upYours 964520782  8  6 f   -     -     -         9925    0    0 
_7upYours 964520804  8  5 f   -     -     -         9925    0    0 
_7upYours 964520821  8  4 f   -     -     -         9925    0    0 
_7upYours 964520834  8  3 f   -     -     -         9925    0    0 
_7upYours 964520878  8  2 Bf  -     -     -         9925   50    0 
_7upYours 964520890  8  1 Bf  -     -     -         9875   25    0 
_7upYours 964520911  8  8 f   -     -     -         9850    0    0 
_7upYours 964520958  8  7 f   -     -     -         9850    0    0 
_7upYours 964520982  8  6 f   -     -     -         9850    0    0 
_7upYours 964520997  8  5 f   -     -     -         9850    0    0 
_7upYours 964521004  8  4 f   -     -     -         9850    0    0 
_7upYours 964521021  8  3 f   -     -     -         9850    0    0 
_7upYours 964521039  8  2 Bk  kf    -     -         9850  100    0 
_7upYours 964521045  8  1 Bf  -     -     -         9750   50    0 
_7upYours 964521066  8  8 f   -     -     -         9700    0    0 
_7upYours 964521076  8  7 f   -     -     -         9700    0    0 
_7upYours 964521093  8  6 f   -     -     -         9700    0    0 
_7upYours 964521097  8  5 f   -     -     -         9700    0    0 
_7upYours 964521112  8  4 f   -     -     -         9700    0    0 
_7upYours 964521134  7  2 f   -     -     -         9700    0    0 
_7upYours 964521161  7  2 Bk  k     k     f         9700  100    0 
_7upYours 964521178  7  1 Bf  -     -     -         9600   50    0 
_7upYours 964521201  7  7 f   -     -     -         9550    0    0 
_7upYours 964521227  7  6 f   -     -     -         9550    0    0 
_7upYours 964521232  7  5 f   -     -     -         9550    0    0 
_7upYours 964521242  7  4 f   -     -     -         9550    0    0 
_7upYours 964521255  7  3 f   -     -     -         9550    0    0 
_7upYours 964521299  7  2 Bf  -     -     -         9550  200    0 
_7upYours 964521324  7  1 Bf  -     -     -         9350  100    0 
_7upYours 964521340  7  7 f   -     -     -         9250    0    0 
_7upYours 964521370  7  6 f   -     -     -         9250    0    0 
_7upYours 964521407  7  5 f   -     -     -         9250    0    0 
_7upYours 964521428  7  4 f   -     -     -         9250    0    0 
_7upYours 964521446  7  3 f   -     -     -         9250    0    0 
_7upYours 964521454  7  2 Bf  -     -     -         9250  200    0 
_7upYours 964521466  7  1 Bf  -     -     -         9050  100    0 
_7upYours 964521494  7  7 f   -     -     -         8950    0    0 
_7upYours 964521498  7  6 f   -     -     -         8950    0    0 
_7upYours 964521502  7  5 f   -     -     -         8950    0    0 
_7upYours 964521514  7  4 f   -     -     -         8950    0    0 
_7upYours 964521533  7  3 f   -     -     -         8950    0    0 
_7upYours 964521551  7  2 Bf  -     -     -         8950  200    0 
_7upYours 964521587  7  1 Bf  -     -     -         8750  100    0 
_7upYours 964521601  7  7 f   -     -     -         8650    0    0 
_7upYours 964521617  6  5 f   -     -     -         8650    0    0 
_7upYours 964521646  6  4 f   -     -     -         8650    0    0 
_7upYours 964521648  6  3 f   -     -     -         8650    0    0 
_7upYours 964521662  6  2 Bk  k     k     kf        8650  400    0 
_7upYours 964521681  6  1 Bc  k     k     k         8250  400    0 8h As
_7upYours 964521703  6  6 f   -     -     -         7850    0    0 
_7upYours 964521713  6  5 f   -     -     -         7850    0    0 
_7upYours 964521718  6  4 rc  kf    -     -         7850 4800    0 
_7upYours 964521738  6  3 f   -     -     -         3050    0    0 
_7upYours 964521749  6  2 Bk  k     k     b         3050 1400 2000 
_7upYours 964521766  6  1 Br  -     -     -         3650 1200 1600 
_7upYours 964521771  6  6 c   f     -     -         4050 1400    0 
_7upYours 964521780  6  5 f   -     -     -         2650    0    0 
_7upYours 964521786  6  4 f   -     -     -         2650    0    0 
_7upYours 964521807  6  3 f   -     -     -         2650    0    0 
_7upYours 964521831  6  2 Bc  bA    -     -         2650 2650    0 7c 2h
_7upYours 964525898  6  3 f   -     -     -        10000    0    0 
_7upYours 964525928  6  2 Bf  -     -     -        10000   50    0 
_7upYours 964525943  6  1 Bf  -     -     -         9950   25    0 
_7upYours 964525957  6  6 f   -     -     -         9925    0    0 
_7upYours 964525979  6  5 f   -     -     -         9925    0    0 
_7upYours 964525986  6  4 f   -     -     -         9925    0    0 
_7upYours 964525995  6  3 f   -     -     -         9925    0    0 
_7upYours 964526032  6  2 Bf  -     -     -         9925   50    0 
_7upYours 964526047  6  1 Bf  -     -     -         9875   25    0 
_7upYours 964526063  6  6 f   -     -     -         9850    0    0 
_7upYours 964526079  6  5 f   -     -     -         9850    0    0 
_7upYours 964526084  6  4 f   -     -     -         9850    0    0 
_7upYours 964526153  5  3 f   -     -     -         9850    0    0 
_7upYours 964526173  5  2 B   -     -     -         9850   50   75 
_7upYours 964526188  5  1 Bf  -     -     -         9875   25    0 
_7upYours 964526205  5  5 r   -     -     -         9850  600  850 
_7upYours 964526215  5  4 f   -     -     -        10100    0    0 
_7upYours 964526222  5  3 f   -     -     -        10100    0    0 
_7upYours 964526238  5  2 Br  -     -     -        10100  150  200 
_7upYours 964526252  5  1 Bf  -     -     -        10150   25    0 
_7upYours 964526260  5  5 r   -     -     -        10125  350  500 
_7upYours 964526269  5  4 f   -     -     -        10275    0    0 
_7upYours 964526282  5  3 f   -     -     -        10275    0    0 
_7upYours 964526306  5  2 Bf  -     -     -        10275  100    0 
_7upYours 964526314  5  1 Br  -     -     -        10175  400  600 
_7upYours 964526324  5  5 f   -     -     -        10375    0    0 
_7upYours 964526330  5  4 f   -     -     -        10375    0    0 
_7upYours 964526348  5  3 f   -     -     -        10375    0    0 
_7upYours 964526372  5  2 Bk  k     b     -        10375  500  800 
_7upYours 964526418  5  1 Bc  kc    k     b        10675 1300 1800 
_7upYours 964526460  5  5 f   -     -     -        11175    0    0 
_7upYours 964526477  5  4 f   -     -     -        11175    0    0 
_7upYours 964526482  5  3 f   -     -     -        11175    0    0 
_7upYours 964526520  5  2 Bf  -     -     -        11175  100    0 
_7upYours 964526531  5  1 Bc  b     b     -        11075 1300 1800 
_7upYours 964526569  5  5 f   -     -     -        11575    0    0 
_7upYours 964526582  5  4 f   -     -     -        11575    0    0 
_7upYours 964526590  5  3 f   -     -     -        11575    0    0 
_7upYours 964526600  5  2 Bk  b     -     -        11575  300  400 
_7upYours 964526614  5  1 Bf  -     -     -        11675   50    0 
_7upYours 964526619  5  5 f   -     -     -        11625    0    0 
_7upYours 964526624  5  4 r   -     -     -        11625  700 1000 
_7upYours 964526630  5  3 f   -     -     -        11925    0    0 
_7upYours 964526636  5  2 Bk  k     b     b        11925 3200    0 Kc As
_7upYours 964526673  5  1 Bf  -     -     -         8725  100    0 
_7upYours 964526716  5  5 f   -     -     -         8625    0    0 
_7upYours 964526748  5  4 r   b     b     bA        8625 8625 15075 
_7upYours 964526785  5  3 f   -     -     -        15075    0    0 
_7upYours 964526799  5  2 Bf  -     -     -        15075  200    0 
_7upYours 964526815  5  1 Bc  kf    -     -        14875  700    0 
_7upYours 964526830  5  5 f   -     -     -        14175    0    0 
_7upYours 964526856  5  4 f   -     -     -        14175    0    0 
_7upYours 964526865  5  3 f   -     -     -        14175    0    0 
_7upYours 964526882  5  2 Bc  kf    -     -        14175  700    0 
_7upYours 964526907  5  1 Bf  -     -     -        13475  100    0 
_7upYours 964526922  5  5 f   -     -     -        13375    0    0 
_7upYours 964526925  5  4 f   -     -     -        13375    0    0 
_7upYours 964526929  5  3 f   -     -     -        13375    0    0 
_7upYours 964526942  5  2 Bc  r     -     -        13375 5400 7200 
_7upYours 964526956  5  1 Bf  -     -     -        15175  100    0 
_7upYours 964526969  5  5 f   -     -     -        15075    0    0 
_7upYours 964526993  5  4 f   -     -     -        15075    0    0 
_7upYours 964527015  5  3 f   -     -     -        15075    0    0 
_7upYours 964527025  5  2 Br  -     -     -        15075 1200 1600 
_7upYours 964527036  5  1 Bc  kf    -     -        15475 1400    0 
_7upYours 964527051  5  5 f   -     -     -        14075    0    0 
_7upYours 964527057  5  4 f   -     -     -        14075    0    0 
_7upYours 964527064  5  3 f   -     -     -        14075    0    0 
_7upYours 964527078  5  2 Bf  -     -     -        14075  400    0 
_7upYours 964527095  5  1 Bf  -     -     -        13675  200    0 
_7upYours 964527113  5  5 c   b     -     -        13475 1000 2000 
_7upYours 964527160  5  4 f   -     -     -        14475    0    0 
_7upYours 964527185  5  3 f   -     -     -        14475    0    0 
_7upYours 964527198  5  2 B   -     -     -        14475  400  600 
_7upYours 964527215  5  1 Bf  -     -     -        14675  200    0 
_7upYours 964527226  5  5 f   -     -     -        14475    0    0 
_7upYours 964527257  5  4 f   -     -     -        14475    0    0 
_7upYours 964527261  5  3 f   -     -     -        14475    0    0 
_7upYours 964527291  5  2 Bk  k     k     k        14475  400    0 5s Td
_7upYours 964527325  5  1 Bf  -     -     -        14075  200    0 
_7upYours 964527338  5  5 f   -     -     -        13875    0    0 
_7upYours 964527363  5  4 f   -     -     -        13875    0    0 
_7upYours 964527382  5  3 f   -     -     -        13875    0    0 
_7upYours 964527434  5  2 Bf  -     -     -        13875  800    0 
_7upYours 964527442  5  1 Bf  -     -     -        13075  400    0 
_7upYours 964527454  5  5 c   k     b     -        12675 1800 3000 
_7upYours 964527486  5  4 f   -     -     -        13875    0    0 
_7upYours 964527505  5  3 f   -     -     -        13875    0    0 
_7upYours 964527519  5  2 Bk  c     k     b        13875 7200 14400 2d Ks
_7upYours 964527600  5  1 Bf  -     -     -        21075  400    0 
_7upYours 964527628  5  5 c   f     -     -        20675  800    0 
_7upYours 964527659  5  4 cc  f     -     -        19875 2800    0 
_7upYours 964527688  5  3 f   -     -     -        17075    0    0 
_7upYours 964527706  5  2 Bc  f     -     -        17075 2400    0 
_7upYours 964527721  5  1 Bf  -     -     -        14675  400    0 
_7upYours 964527733  5  5 f   -     -     -        14275    0    0 
_7upYours 964527749  5  4 r   -     -     -        14275 3600 6000 
_7upYours 964527762  5  3 f   -     -     -        16675    0    0 
_7upYours 964527787  5  2 Bf  -     -     -        16675 1600    0 
_7upYours 964527801  5  1 Br  -     -     -        15075 10488 12088 
_7upYours 964527815  5  5 c   b     -     -        16675 3200 4975 7s Ts
_7upYours 964527843  4  4 f   -     -     -        18450    0    0 
_7upYours 964527857  4  3 f   -     -     -        18450    0    0 
_7upYours 964527887  4  2 B   -     -     -        18450 1600 2950 9h 8s
_7upYours 964527916  3  1 Bf  -     -     -        19800  800    0 
_7upYours 964527929  3  3 f   -     -     -        19000    0    0 
_7upYours 964527965  2  2 B   -     -     -        19000 1600 2400 
_7upYours 964527989  2  1 Br  kf    -     -        19800 4800    0 
_7upYours 964528013  2  2 B   -     -     -        15000 1600 2400 
_7upYours 964528017  2  1 BrA -     -     -        15800 15800 17400 
_7upYours 964528029  2  2 Bk  k     k     f        17400 1600    0 
_7upYours 964528048  2  1 Bf  -     -     -        15800  800    0 
_7upYours 964528051  2  2 Bk  f     -     -        15000 1600    0 
_7upYours 964528060  2  1 Bf  -     -     -        13400  800    0 
_7upYours 964528063  2  2 Bk  k     b     k        12600 4800 9600 9d Ad
_7upYours 964528090  2  1 Br  krA   -     -        17400 17400 31800 
_7upYours 964528112  2  2 B   -     -     -        31800 3200 4800 
_7upYours 964528116  2  1 Bf  -     -     -        33400 1600    0 
_7upYours 964528119  2  2 Bk  k     k     b        31800 9600 12800 
_7upYours 964528142  2  1 Bf  -     -     -        35000 1600    0 
_7upYours 964528145  2  2 B   -     -     -        33400 3200 4800 
_7upYours 964528149  2  1 Bf  -     -     -        35000 1600    0 
_7upYours 964528153  2  2 Bf  -     -     -        33400 3200    0 
_7upYours 964528158  2  1 Bf  -     -     -        30200 1600    0 
_7upYours 964528160  2  2 Bk  f     -     -        28600 3200    0 
_7upYours 964528168  2  1 Br  -     -     -        25400 9600 12800 
_7upYours 964528176  2  2 Bf  -     -     -        28600 3200    0 
_7upYours 964528179  2  1 Br  k     k     k        25400 9600    0 2c 2h
_7upYours 964528198  2  2 Bk  f     -     -        15800 3200    0 
_7upYours 964528212  2  1 Bf  -     -     -        12600 1600    0 
_7upYours 964528215  2  2 Bf  -     -     -        11000 3200    0 
_7upYours 964528220  2  1 BrA -     -     -         7800 7800    0 7d Jd
