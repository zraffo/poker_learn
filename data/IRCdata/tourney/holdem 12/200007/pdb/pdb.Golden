Golden    963327662 18 12 cc  kf    -     -        10000  974    0 
Golden    963327783 17 11 c   k     kf    -         9026  150    0 
Golden    963327846 17 10 f   -     -     -         8876    0    0 
Golden    963327908 17  9 c   b     c     cA        8876 8876    0 7s 5d
